### ISSUE TITLE
Misleading success message

### ISSUE THREAD

**Issue by jekuno** [svenfuchs/gem-release#60]
When running `gem release` a success message gets printed even if the release didn't succeed (I disconnected from internet for testing purposes).

Check this line:
https://github.com/svenfuchs/gem-release/blob/06106d11d6d31f013e55e16894c7944f6440f856/lib/rubygems/commands/release_command.rb#L43

**Comment by svenfuchs** [comment]
This should be fixed in `2.0.0.pre.1`, see https://github.com/svenfuchs/gem-release/pull/64

```
$ gem release
Releasing foo with version 0.0.5

Building foo.gemspec
$ gem build foo.gemspec
  Successfully built RubyGem
  Name: foo
  Version: 0.0.5
  File: foo-0.0.5.gem

Pushing foo-0.0.5.gem
$ gem push foo-0.0.5.gem --key localhost --host http://localhost:9292
Pushing gem to http://localhost:9292...

Deleting left over gem file foo-0.0.5.gem
$ rm -f foo-0.0.5.gem
ERROR:  While executing gem ... (Errno::ECONNREFUSED)
    Failed to open TCP connection to localhost:9292 (Connection refused - connect(2) for "localhost" port 9292)
```

**Comment by svenfuchs** [comment]
Thanks for reporting, @jekuno! I'm going to close this here.
